src/main/java/frc/robot/subsystems/climb/Climb.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.subsystems.climb;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.ports;
+import frc.robot.subsystems.UnitModel;
+
+public class Climb extends SubsystemBase {
+
+    private TalonSRX rightMaster = new TalonSRX(ports.Drivetrain.RIGHT_MASTER);
+    private VictorSPX rightSlave = new VictorSPX(ports.Drivetrain.RIGHT_SLAVE);
+    private TalonSRX leftMaster = new TalonSRX(ports.Drivetrain.LEFT_MASTER);
+    private VictorSPX leftSlave = new VictorSPX(ports.Drivetrain.LEFT_SLAVE);
+    private UnitModel UnitModel = new UnitModel(Constants.Drivetrain.TICKS_TO_METER);
+
+
+
+    public Climb(){

```

Comment:
If you set the sensor phase, attach the sensor to the motor. You can't change something that the system doesn't exists